.vscode/settings.json

```diff
@@ -22,11 +22,10 @@
     "rust-analyzer.cargo.unsetTest": [
         "api",
         "common",
-        "agent",
-        "server",
+        "ank-agent",
+        "ank-server",
```

Comment:
Why is it renamed, I thought it must match the paket name?

Reply:
It must match to the name configured in the `Cargo.toml`.

Reply:
Ok 👍